### PR TITLE
Cherry-pick: Update packaging version for 6.2

### PR DESCRIPTION
**Pull request by stanleytsang-amd**
Do not merge until PM approval given.

### CODE REVIEW

**Comment by stanleytsang-amd** [comment]
@umfranzw @RobsonRLemos Hi can we get codeowner review and approval on this PR? Thanks. Failure is due to CI issues.  PM approval was given this morning, should be able to merge this in soon.

**Review by RobsonRLemos — APPROVED** [review]
LGTM. Failure is due to CI issues.

**Pull request merged**
